lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -1263,25 +1264,17 @@ public long currentTimeMillis() {
         return System.currentTimeMillis();
     }
 
-    private String getRequestDescription(LockRequest request) {
-        StringBuilder builder = new StringBuilder();
-        builder.append("\twaiting to lock() ").append(request);
-        builder.append(" starting at ").append(ISODateTimeFormat.dateTime().print(DateTime.now()));
-        builder.append("\n\tfor requesting thread\n\t\t");
-        builder.append(request.getCreatingThreadName()).append("\n");
-        return builder.toString();
-    }
-
-    private String updateThreadName(LockRequest request) {
+    private static String updateThreadName(LockRequest request) {
         String currentThreadName = Thread.currentThread().getName();
-        String requestDescription = getRequestDescription(request);
-        tryRenameThread(currentThreadName + "\n" + requestDescription);
```

Comment:
If desired, I can pull this refactor tweak out to a separate PR, but I wanted fix up to allow Indy string concat for this code path.